lib/cambiatus/error.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Cambiatus.Error do
+  @moduledoc """
+  Standard error structure to be used in Cambiatus.
+
+  Used to represent and display operation statuses in a more standard way
+
+  Errors have two fields:
+  * `type`: atom representing the possible error types
+  * `message`: strings with the error message
+  """
+  require Logger
+
+  alias Cambiatus.Error
+
+  defstruct [:type, :message, :x]
+
+  @doc """
+  Converts common error structures used on libraries such as Ecto, Absinthe to `Cambiatus.Error`
+  """
+  def from(%Ecto.Changeset{valid?: false} = changeset) do
+    changeset
+    |> Ecto.Changeset.traverse_errors(fn {err, opts} ->
+      Enum.reduce(opts, err, fn {key, value}, acc ->
+        String.replace(acc, "%{#{key}}", to_string(value))
+      end)
+    end)
+  end
+
+  def from(other) do
+    Logger.error("Unhandled error term: \n#{inspect(other)}")
+    Sentry.capture_message("Sign up failed", extra: %{error: other})
```

Comment:
Looks like this line should be more general than `"Sign up failed"`. Maybe like this?

```suggestion
    Sentry.capture_message("Unhandled error term:", extra: %{error: other})
```

Reply:
ops, thanks for noticing bro 😀